Automattic/wp-calypso
# Subscribers: Paid subscribers content importer UI fixes

Related to https://github.com/Automattic/wp-calypso/issues/93150

## Proposed Changes

It:
- adds "external" icon for "Export content" and "Export subscribers" links
- hides the upload description
- hides duplicated action buttons
- fixes action buttons spacing for "Content", "Subscribers" and "Paid Subscribers" step

| Before | After |
|--------|--------|
| <img width="733" alt="Screenshot 2024-08-09 at 16 27 32" src="https://github.com/user-attachments/assets/5b087bc6-2dec-4e0d-890b-c9859f3f9ef8"> | <img width="733" alt="Screenshot 2024-08-09 at 16 26 42" src="https://github.com/user-attachments/assets/81952321-b784-48a4-973b-1988577a7d14"> |

## Why are these changes being made?

To make content importer UI look closer to the designs.

## Testing Instructions



1. Go to `/import/newsletter/substack/{blogHostname}`
2. Type the Substack blog URL into the input and click "Continue"
3. Make sure the "Content" step looks as expected
4. Go to Tools -> Import -> Substack and make sure it doesn't introduce any regression to the existing importer

## Pre-merge Checklist
